ci-operator/step-registry/idp/README.md

```diff
@@ -0,0 +1,70 @@
+# Identity Providers (IDP) Configuration for OpenShift ProwCI
```

Comment:
Not sure if I'm correct, I only saw "Prow CI" spelling, not "ProwCI".

Reply:
Yes, you are right,  it's "Prow CI",  Let me update the that, Thanks